sentences/03-text-2.yaml

```diff
@@ -39,36 +39,36 @@ sentences:
   zh-hans: 你好！你也是老师吗？
   pinyin: Nǐ hǎo! Nǐ yěshì lǎoshī ma?
   en: Hello! Are you also a teacher?
-  es: Mucho gusto. ¿Tú también eres profesor?
+  es: Hola. ¿Tú también eres profesor?
 - id: f7bb1270-498f-480c-a057-4d4c053ec248
   zh-hans: 您好！我不是老师。我是医生。
   pinyin: Nín hǎo! Wǒ bùshì lǎoshī. Wǒ shì yīshēng.
   en: Hello! I am not a teacher. I am a doctor.
-  es: ¡No soy profesor! No soy profesor. Soy médico.
+  es: ¡Hola! No soy profesor. Soy médico.
 - id: 2fd4a31d-37cb-4323-b241-d9b24a67facd
   zh-hans: 力波，这是你奶奶吗？
   pinyin: Lìbō, zhè shì nǐ nǎinai ma?
   en: Libo, is this your paternal grandmother?
-  es: Libo, ¿es tu abuela?
+  es: Libo, ¿es tu abuela de lado paterno?
 - id: 12f99a4c-19c0-4461-8e7b-58347abc93c1
   zh-hans: 不是，她是我外婆。
   pinyin: Bùshì, tā shì wǒ wàipó.
   en: No. She is my maternal grandmother.
-  es: No, es mi abuela.
+  es: No, es mi abuela de lado materno.
```

Comment:
Good job! This conforms with [convention no. 1](https://github.com/rdrg109/subtitles-npcr/blob/main/docs/conventions.org#1-prioritize-conveying-of-full-meaning-over-naturality-of-speech).